### PR TITLE
metal pipe sound when body falls

### PR DESCRIPTION
**Pull request by Slava0135**
## Media

https://github.com/space-wizards/space-station-14/assets/40753025/9a068857-3617-4f6d-8b85-f2c653b243d1

## Breaking changes

**Changelog**

:cl:
- tweak: Bodies have a chance to play a metal pipe sound when falling

### CODE REVIEW

File: Resources/Audio/Effects/attributions.yml [hunk 1]
```diff
@@ -216,3 +216,8 @@
   copyright: Bhijn and Myr (github username deathride58)
   license: CC0-1.0
   source: https://github.com/space-wizards/space-station-14/pull/23212
+
+- files: ["metal_pipe_falling.ogg"]
+  license: "CC0-1.0"
+  copyright: "jixaw"
+  source: "https://www.youtube.com/watch?v=iDLmYZ5HqgM"
```
Comment (FairlySadPanda) [inline Resources/Audio/Effects/attributions.yml:223]: It's a stock sound from prior to 1993 with unknown provenance. I wanted to add it in as a round restart sfx :(

**Comment by Slava0135** [comment]
https://youtu.be/cEf_hfDgIBo?si=FHrS7O-FbzyM3Tgg
Real source is unknown but according to this, it appeared in many games before it became popular in 2020. I think we are safe to use this (at least for april 1). Any thoughts?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
1. That video doesn't even have a license nor is it the original source.
2. I doubt we can use this.

**Comment by PJB3005** [comment]
Re-open when you can find the original sound CD from the 90s this came from lol.

**Comment by Slava0135** [comment]
i think i found source for it

image

https://soundeffects.fandom.com/wiki/Sound_Ideas,_METAL,_DROP_-_METAL,_DROP_METAL_POLE:_LOUD_DROP_AND_BOUNCE_02

on their site it says its Roalty Free, though you need to buy it???
image
https://www.sound-ideas.com/Product/198/L2-Sound-Effects-Library-by-Frank-Serafine

**Comment by Slava0135** [comment]
it was modified though (by Jixaw), they uploaded it on their second channel (without license though)
https://www.youtube.com/watch?v=kPWv44v0b04
image

i wouldn't merge this into master, but for april 1st, i think its fine, if we just put link to this video (and i actually use audio file from there)

**Comment by Slava0135** [comment]
this though https://www.sound-ideas.com/Page/Sound-Ideas-End-User-License
image

**Comment by Slava0135** [comment]
Assuming original sound comes from Sound Ideas, you need to buy license, so we are not allowed to use it.

I think they just don't care about videos
image

**Pull request closed without merging**